src/tool/cppwinrt/strings/base_string.h

```diff
@@ -289,10 +289,9 @@ namespace winrt::impl
         }
     };
 
-    inline _locale_t get_default_locale()
+    inline handle_type<locale_handle_traits> get_default_locale()
```

Comment:
Can we remove this function now?

Reply:
Ah, yeah. Good point. I think this may also remove our dependency on `<clocale>`.